src/js/events/CosmosPackagesActions.js

```diff
@@ -149,6 +150,35 @@ const CosmosPackagesActions = {
     });
   },
 
+  fetchPackageVersions(packageName) {
+    // necessary for the backend
+    // even though it's not in use
+    const includePackageVersions = false;
+
+    RequestUtil.json({
+      contentType: getContentType("list-versions", "request", "v1"),
+      headers: { Accept: getContentType("list-versions", "response", "v1") },
+      method: "POST",
+      url: `${Config.rootUrl}${Config.cosmosAPIPrefix}/list-versions`,
+      data: JSON.stringify({ packageName, includePackageVersions }),
```

Comment:
What happens if `includePackageVersions` is not included in the request?

Reply:
the server denies the request and returns an error.